.github/workflows/publish.yml

```diff
@@ -0,0 +1,68 @@
+name: Publish to npmjs
+
+on:
+  # Who has permission to workflow_dispatch
+  # https://github.com/orgs/community/discussions/26622
+  workflow_dispatch:
```

Comment:
` if: contains('["..."]', github.actor)` does not really makes sense to me

Reply:
`workflow_dispatch` is a slightly recent feature iirc; maybe they couldn't use the feature at that moment.
I agree with using `workflow_dispatch` :+1: